src/debugger/config.clj

```diff
@@ -1,5 +1,5 @@
 (ns debugger.config
-  (:require [clj-time.core :as t]))
+  (:require [debugger.utils :as t]))
```

Comment:
Maybe `t` should be changed to `u`. Looks kind of add like this. :-) Or `debugger.utils` can be `debugger.time`?